README.rst

```diff
@@ -7,67 +7,32 @@ Tools for the Piksi GNSS receiver
 
-Python tools for the Piksi GNSS receiver. This repository includes a
-Swift console UI application, as well as a variety of command line
-utilities (firmware bootloader, serial port data logging, etc.).
+Python tools for the Piksi GNSS receiver. This repository includes a a variety
+of command line utilities (firmware bootloader, serial port data logging,
+etc.).
 
 Setup
 -----
 
-It is advised to install ``piksi_tools`` inside a virtualenv to avoid modifying
-global system state.  To setup a python 3.5 virtual environment first install
-python 3.5 via your package manager if needed, and then run::
+It is advised to install ``piksi_tools`` inside a conda environment to avoid modifying
+global system state.  To setup a conda environment first install
+Miniconda via your package manager if needed, and then run::
 
-  virtualenv -p python3.5 ~/py3
-  source ~/py3/bin/activate
+  conda create -n piksi_tools python=3.7
+  conda activate piksi_tools
 
-To install the dependencies for the basic tools: ``pip install -r requirements.txt``
-
-To install the dependencies for the console GUI, run ``make deps``. Besides
-system packages, this also installs python dependencies into the current
-(virtual) environment, and includes the deps for the aforementioned basic tools.
-
-Finally, ``pip install -e .`` to set up a dev install in the local dev environment.
-
-To run the installed console from the current env, use ``python -m piksi_tools.console.console``
+Run ``pip install -e .[test]`` to set up a dev install in the local dev environment.
 
 Python version support
 ~~~~~~~~~~~~~~~~~~~~~~
 
 * The most important command line tools - ``bootload_v3.py``, ``fileio.py``,
-  ``serial_link.py``, and ``settings.py`` - support Python 2.7, 3.5, and 3.7
-
-* Console GUI under Linux supports 2.7 and 3.5. Experimental support for Python 3.7
-
-* Console GUI under MacOS and Windows is tested against Python 3.5 but probably
-  would support all versions that Linux GUI supports, but some of those might
-  need a manual GUI backend change/installation
-
-* Pre-built (pyinstaller) binaries for most platforms use Python 3.5.  On Ubuntu,
-  Python 3.6 is used.
-
-Usage Examples
---------------
-
-Console example
-~~~~~~~~~~~~~~~
-
-To just use the Swift console, download binary installers for Windows and OS X.
-
-  Latest console for `Piksi Multi <http://downloads.swiftnav.com/swift_console>`__
-
-  Older versions of console for use with `Piksi v2 <http://downloads.swiftnav.com/piksi_console>`__
-
-  For x86-64 Linux, tar.gz distributions are available for Swift Console `Piksi Multi <http://downloads.swiftnav.com/swift_console>`__
-
-To run the console GUI from the command line, install dependencies and run ``PYTHONPATH=. python piksi_tools/console/console.py``.
-
-For command line arguments, see `console.py <https://github.com/swift-nav/piksi_tools/blob/master/piksi_tools/console/console.py>`__
+  ``serial_link.py``, and ``settings.py`` - support Python 3.7 onward.
```

Comment:
`libsettings` is also being recommended over `settings.py` in recent time

Reply:
Well, piksi_tools.settings uses libsettings, but we could probably move the settings module into libsettings and remove the one from here... the problem is the two libs are somewhat interwoven since libsettings would need to gain support to talk over serial/tcp (which, in example code, it circularly uses piksi_tools for...).

Reply:
swiftcar still uses `piksi_tools.settings`